kefir.js.flow

```diff
@@ -22,17 +22,17 @@ export type Subscription = {
 };
 
 export type Observer<V,E> = {
-  value?: (value: V) => void;
-  error?: (err: E) => void;
-  end?: () => void;
+  +value?: (value: V) => void;
+  +error?: (err: E) => void;
+  +end?: () => void;
 };
 
 declare class Observable<+V,+E=*> {
   toProperty(): Property<V,E>;
   toProperty<V2>(getCurrent: () => V2): Property<V|V2,E>;
   changes(): Observable<V,E>;
 
-  observe(obs: Observer<V,E>): Subscription;
+  observe(obs: Observer<V,E>, ...none: empty[]): Subscription;
```

Comment:
Maybe we should also change order of overloads in addition (or instead) to this? To support `obs.observe(x => { /* my on value handler */ })` case.